src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -23,7 +23,7 @@ class Webui::WebuiController < ActionController::Base
   before_action :current_announcement, unless: -> { request.xhr? }
   before_action :fetch_watchlist_items
   before_action :set_paper_trail_whodunnit
-  before_action :set_unread_notifications
+  before_action :set_unread_notifications, unless: -> { params[:controller] == 'webui/users/notifications' && params[:action] == 'update' }
```

Comment:
I wouldn't mind renaming both the action name and the shared variable to `set_unread_notifications_count` and `@unread_notifications_count`, otherwise I would expect to have the collection, not the number.
But it can also be done later on in a different PR.

Reply:
I guess you where looking for [`skip_before_action`](https://api.rubyonrails.org/classes/AbstractController/Callbacks/ClassMethods.html#method-i-skip_before_action)

Reply:
Arg... true. I'll fix that. Thanks!